### DISCUSSION TITLE
File Structure Question

### DISCUSSION THREAD

**Discussion by mcharlee** [cameroncros/OctoPrint-DiscordRemote#236] [Q&A]
Hey so I noticed when using the "files" command the path output only contains the immediate parent of the file. For example, "Folder/File.gcode" I have an organization system that is one level deeper on my printer that seems to break the commands. I have "User_Folder/Folder/File.gcode". When attempting to use the "print" command with both "Folder/File.gcode" and "User_Folder/Folder/File.gcode" I receive a message "Failed to find the file".  I also tested a file at the "User_Folder/File.gcode" location and it was successful so I know that the commands work on my system, but it seems that the path is too complicated. My questions, Have I missed a step in the setup that would solve this problem? Is there a reason the path is limited to immediate parent and file? and Is there a way to make the plugin compatible with different file structures?

**Comment by cameroncros** [comment]
I barely understand the file structure, and it could vary depending on the octoprint install. I believe it should work though with folders and sub-folders, and not working is probably a bug.
